Add error handling reference to README (#37)
IBM/cloudant-go-sdk: README.md

You have deleted the document.
```

## Error handling

> To see sample code, how we suggest to handle errors, please check [Cloudant API docs](https://cloud.ibm.com/apidocs/cloudant?code=go#error-handling)

Reviewer: ```suggestion
To see sample code, how we suggest handling errors, please check [Cloudant API docs](https://cloud.ibm.com/apidocs/cloudant?code=go#error-handling)
```

Reviewer 2: This can be improved to:
```suggestion
For sample code on handling errors, please see [Cloudant API docs](https://cloud.ibm.com/apidocs/cloudant?code=go#error-handling)
```
